docroot/themes/humsci/su_humsci_theme/scss/components/organisms/_hs-person__node.scss

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
Lets move this to the node specific style sheet in `docroot/themes/humsci/su_humsci_theme/scss/nodes/hs_person.scss`